src/modelinstance.cpp

```diff
@@ -80,25 +80,11 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
         }
     }
     this->inputsInfo.clear();
+
     for (const auto& pair : networkInputs) {
         const auto& name = pair.first;
         auto input = pair.second;
-
-        // Data from network
-        auto precision = input->getPrecision();
-        auto layout = input->getLayout();
         auto shape = input->getTensorDesc().getDims();
-
-        // Data from config
-        if (config.getLayout().size()) {
-            // Single layout for all inputs
-            layout = TensorInfo::getLayoutFromString(config.getLayout());
-        } else if (config.getLayouts().count(name)) {
```

Comment:
You are removing dead code from our source regarding layout switch. There might be more places with layout dead code. Have you checked that?

Reply:
Ok I see, you do not remove but move it.